WebClock/Controllers/ClockController.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Mvc;
+using Newtonsoft.Json;
+
+namespace WebClock.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class ClockController : Controller
+    {
+        [HttpGet]
+        public JsonResult GetActualDate()
+        {
+            return Json(new { CurrentDate = DateTime.Now });
```

Comment:
Z tego co pamiętam rozmawialiśmy o aktualnej godzinie, nie o dacie

Reply:
poprawione na wyswietlanie godziny,minut i sekund

Reply:
Można też zdefiniować konkretną klasę (np. Dto) której obiekt będziemy zwracali. Nazwa udokumentuje nam dane które wysyłamy (obiekt anonimowy tego nie robi)

Reply:
Można, ale myślę, że to jest bardzo prosty obiekt, więc byłoby to nadmiarowe.

Reply:
Uważam, że Rafał ma tu rację, bo obiekt stał się już bardziej skomplikowany i przypuszczam, że przy serializacjach, deserializacjach, porównywaniu do testów itp. przyda się jawny typ zamiast anonimowego. IMHO pracę na obiektach anonimowych powinniśmy ograniczyć do minimum, a szczególnie nie używać w komunikacji pomiędzy modułami, obiektami, a szczególnie w API.

Reply:
Okej, przekonaliście mnie. Zmieniliśmy to :)

Reply:
super

Reply:
lapka-w-gore ;)